### PR TITLE
chore(slack): default message builder to block kit

### PR DESCRIPTION
**Pull request by cathteng**
Default the Slack message builder to use block kit. This touches a lot of file paths so a ton of tests needed to be deleted (if they were testing non-block kit) or slightly refactored.

The only non-test file changed is https://github.com/getsentry/sentry/pull/68016/files#diff-8d9c86c08a539227a52e53d0bb1cb1bc1048b8875b6dce664f3114453fd042ea

### CODE REVIEW

File: src/sentry/integrations/slack/message_builder/issues.py [hunk 1]
```diff
@@ -265,36 +264,20 @@ def get_context(group: Group) -> str:
...
 def get_option_groups_block_kit(group: Group) -> Sequence[Mapping[str, Any]]:
     all_members = group.project.get_members_as_rpc_users()
     members = list({m.id: m for m in all_members}.values())
     teams = group.project.teams.all()
-    use_block_kit = features.has("organizations:slack-block-kit", group.project.organization)
 
     option_groups = []
     if teams:
-        team_options = format_actor_options(teams, use_block_kit)
+        team_options = format_actor_options(teams, True)
```
Comment (ykamo001) [inline src/sentry/integrations/slack/message_builder/issues.py:274]: next PR we can remove the second boolean parameter right?

Comment (cathteng) [reply src/sentry/integrations/slack/message_builder/issues.py:274]: yes

Comment (cathteng) [reply src/sentry/integrations/slack/message_builder/issues.py:274]: i'm trying to isolate the changes to this one file, but it ended up affecting 123098 tests

File: src/sentry/integrations/slack/message_builder/issues.py [hunk 2]
```diff
@@ -425,91 +408,51 @@ def build_actions(
...
             name="assign",
             label="Select Assignee...",
             type="select",
-            selected_options=format_actor_options([assignee]) if assignee else [],
-            option_groups=(
-                get_option_groups(group)
-                if not use_block_kit
-                else get_option_groups_block_kit(group)
-            ),
+            selected_options=format_actor_options([assignee], True) if assignee else [],
```
Comment (ykamo001) [inline src/sentry/integrations/slack/message_builder/issues.py:445]: we added a true here in `format_action_options`, was that intentional?

**Review by ykamo001 — APPROVED** [review]
some comments, but looks great! Thanks for the clean up! Just double checking the boolean param

**Pull request merged**